clusters/lib/snmp-exporter/application.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: snmp-exporter
+  labels:
+    nerc.mghpcc.org/sync-policy: common
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/ocp-on-nerc/nerc-ocp-config.git
+    targetRevision: HEAD
+    path: REPLACEME
```

Comment:
You need to replace this value in a patch. Right now, the value `REPLACEME` will show up in the output of `kustomize build` and this application will fail to apply.

Reply:
@larsks Oops, thanks, you're right, I replaced the `opentelemetry` patch with `nmp-exporter`. I have fixed this patch.